### PR TITLE
first Version

### PR DESCRIPTION
**Pull request by eyenx**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM alpine:latest
+
+LABEL org.opencontainers.image.authors="Adfinis AG <https://adfinis.com>"
```
Comment (hairmare) [inline Dockerfile:3]: Let's add some more labels... Imma get back to these tomorrow.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+RUN apk add --no-cache \ 
+    bash \
+    jq \
+    curl \
+    tcpdump \
+    wget \
+    openssl \
+    bind-tools
```
Comment (hairmare) [inline Dockerfile:12]: Can you be nice and sort these alphabetically?

Comment (eyenx) [reply Dockerfile]: 63886a6

Comment (eyenx) [reply Dockerfile]: 2d28f41

**Comment by eyenx** [comment]
> Can we move `Dockerfile` to `alpine/Dockerfile` from the get-go?
Okay.
> You might as well also get started on CD, do we want to leverage https://github.com/docker/build-push-action?
No. We did not do it on back8sup, we shouldn't do it here. We also would need a service account or something to push to `adfinissygroup`.

We should have one strategy to push to docker hub. Auto builds or GH Workflows. Not both.

File: Dockerfile (hunk 1 above)
Comment (hairmare) [inline Dockerfile:3]: Maybe something like this?
```suggestion
LABEL org.opencontainers.image.title="alpine-debug-container" \
      org.opencontainers.image.description="Alpine based debug container with tools for debugging networking issues and more." \
      org.opencontainers.image.authors="Adfinis AG <https://adfinis.com>" \
      org.opencontainers.image.vendor="Adfinis" \
      org.opencontainers.image.url="https://github.com/adfinis-sygroup/debug-container" \
      org.opencontainers.image.licenses="AGPL-3.0-or-lateræ
```
Most of what's left in https://github.com/opencontainers/image-spec/blob/master/annotations.md is stuff that we can only set at build time.

File: Dockerfile [hunk 3]
```diff
@@ -1,6 +1,11 @@
 FROM alpine:latest
 
-LABEL org.opencontainers.image.authors="Adfinis AG <https://adfinis.com>"
+LABEL org.opencontainers.image.title="alpine-debug-container" \
+      org.opencontainers.image.description="Alpine based debug container with tools for debugging networking issues and more." \
+      org.opencontainers.image.authors="Adfinis AG <https://adfinis.com>" \
+      org.opencontainers.image.vendor="Adfinis" \
+      org.opencontainers.image.url="https://github.com/adfinis-sygroup/debug-container" \
+      org.opencontainers.image.licenses="AGPL-3.0-or-lateræ
```
Comment (hairmare) [inline Dockerfile:8]:
```suggestion
      org.opencontainers.image.licenses="AGPL-3.0-or-later
```
my bad, I'm sorry

Comment (eyenx) [reply Dockerfile:8]:
```suggestion
      org.opencontainers.image.licenses="AGPL-3.0-or-later"
```

**Pull request merged**